### PR TITLE
docs: add quick start guide

### LINKED ISSUE #21: Guide for a quick start is needed
### Problem

There is no description and code example that shows where the middleware must / can be inserted into a Mezzio application.

### Suggestion

The basic usage with adding middleware and creating the configuration can be described in a quick start guide. A new page and navigation entry is needed for this.

### PR DESCRIPTION
**Pull request by boesing**
|    Q          |   A
|-------------- | ------
| Documentation | yes

### Description

Adding a Quick Start Guide to help users to have an idea on how to use this component in their project(s).

### CODE REVIEW

File: docs/book/v1/intro.md [hunk 1]
```diff
@@ -44,3 +44,70 @@ the response is created immediately.
 
 If the route is not explicit, *all* request methods are checked to provide a
 list of possible request methods to the client.
+
+## Quick Start
```
Comment (froschdesign) [inline docs/book/v1/intro.md]: For consistency with other documentations, create a separate page for the quick start. Insert this page in the navigation after the installation page.

File: docs/book/v1/intro.md [hunk 2]
```diff
@@ -44,3 +44,70 @@ the response is created immediately.
...
+
+After installing this component, the `CorsMiddleware` has to be added to the pipeline configuration.
+
+Depending on the configuration style you've chosen for the project, the pipeline might be either an array (in combination with the `ApplicationConfigInjectionDelegator`) or an anonymous function.
+
+> ### Positioning of the `CorsMiddleware` is crucial
```
Comment (froschdesign) [inline docs/book/v1/intro.md]:
```suggestion
> ### Positioning of the `CorsMiddleware` is Crucial
```

File: docs/book/v1/intro.md [hunk 3]
```diff
@@ -44,3 +44,70 @@ the response is created immediately.
...
+> 
+> The `CorsMiddleware` **MUST** be added to the pipeline **before** the `RouteMiddleware`.
+
+### Anonymous function pipeline
```
Comment (froschdesign) [inline docs/book/v1/intro.md]:
```suggestion
### Anonymous Function Pipeline
```

File: docs/book/v1/intro.md [hunk 4]
```diff
@@ -44,3 +44,70 @@ the response is created immediately.
...
+
+```php
+<?php
+
```
Comment (froschdesign) [inline docs/book/v1/intro.md]:
```suggestion
```

File: docs/book/v1/intro.md [hunk 5]
```diff
@@ -44,3 +44,70 @@ the response is created immediately.
...
+    $app->pipe(CorsMiddleware::class);
+    // [...]
+    $app->pipe(RouteMiddleware::class);
+    // [...]
+    $app->pipe(DispatchMiddleware::class);
+    // [...]
+};
+```
+
+### Config injection pipeline
```
Comment (froschdesign) [inline docs/book/v1/intro.md]:
```suggestion
### Config Injection Pipeline
```

File: docs/book/v1/intro.md [hunk 6]
```diff
@@ -44,3 +44,70 @@ the response is created immediately.
...
+
+```php
+<?php
+
```
Comment (froschdesign) [inline docs/book/v1/intro.md]:
```suggestion
```

**Review by froschdesign — CHANGES_REQUESTED** [review]
For headlines the Chicago Manual of Style capitalization rules must be used:

* Capitalize the first and the last word.
* Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
* Lowercase articles (a, an, the), coordinating conjunctions, and prepositions.
* Lowercase the ‘to’ in an infinitive (I want to play guitar).

The online tool [capitalizemytitle.com](https://capitalizemytitle.com/style/Chicago/) can help to capitalize and case convert the headlines.

**Review by froschdesign — APPROVED** [review]

**Comment by boesing** [comment]
@froschdesign I wonder if its somehow possible to verify the Chicago Manual of Style rule with the markdown linter somehow.

**Pull request merged**